src/ansys/aedt/core/maxwell.py

```diff
@@ -2961,6 +2961,132 @@ def assign_zero_tangential_h_field(self, assignment, boundary=None):
+            between conductivity and current density.
+            The default value is ``"10"``.
+        anode_c : str, optional
+            Anode c value that corresponds to the c coefficient in the non-linear relationship
+            between conductivity and current density.
+            The default value is ``"110000000000000"``.
+        anode_d : str, optional
+            Anode d value that corresponds to the d coefficient in the non-linear relationship
+            between conductivity and current density.
+            The default value is ``"2"``.
+        cathode_a : str, optional
+            Cathode a value that corresponds to the a coefficient in the non-linear relationship
+            between conductivity and current density.
+            The default value is ``"300000000"``.
+        cathode_b : str, optional
+            Cathode b value that corresponds to the b coefficient in the non-linear relationship
+            between conductivity and current density.
+            The default value is ``"10"``.
+        cathode_c : str, optional
+            Cathode c value that corresponds to the c coefficient in the non-linear relationship
+            between conductivity and current density.
+            The default value is ``"110000000000000"``.
+        cathode_d : str, optional
+            Cathode d value that corresponds to the d coefficient in the non-linear relationship
+            between conductivity and current density.
+            The default value is ``"2"``.
+
+        Returns
+        -------
+        :class:`ansys.aedt.core.modules.boundary.BoundaryObject`
+            Newly created object. ``False`` if it fails.
+
+        References
+        ----------
+        >>> oModule.AssignResistiveSheet
+
+        Examples
+        --------
+        >>> import ansys.aedt.core
+        >>> from ansys.aedt.core.generic.constants import SOLUTIONS
+        >>> m3d = ansys.aedt.core.Maxwell3d(solution_type="Transient")
+        >>> my_box = m3d.modeler.create_box(origin=[0, 0, 0], sizes=[0.4, -1, 0.8], material="copper")
+        >>> resistive_face = my_box.faces[0]
+        >>> bound = self.aedtapp.assign_resistive_sheet(assignment=resistive_face, resistance="3ohm")
+        >>> self.aedtapp.solution_type = SOLUTIONS.Maxwell3d.Magnetostatic
+        >>> bound = self.aedtapp.assign_resistive_sheet(assignment=resistive_face, non_linear=True)
+        >>> m3d.release_desktop()
+        """
+        if self.solution_type not in ["EddyCurrent", "Transient", "Magnetostatic"]:
+            self.logger.error(
+                "Resistive sheet is applicable only to Eddy current, transient and magnetostatic solvers."
+            )
+            return False
+
+        assignment = self.modeler.convert_to_selections(assignment, True)
+
+        if not name:
+            boundary = generate_unique_name("ResistiveSheet")
+
+        props = dict(
+            {
+                "Faces": assignment,
+            }
+        )
```

Comment:
```suggestion
        props = {
            "Faces": assignment,
        }
```
`{ .. }` is already a dict